web/css/custom/program.scss

```diff
@@ -735,7 +739,39 @@ input[type='number']
 
 #tag-extension-container
 {
-  margin-bottom: 20px;
+  margin-bottom: 1rem;
+
+  > #tags:nth-child(2)
+  {
+    margin-top: 0.5rem;
+  }
+
+  #tags, #extensions
+  {
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+    margin-top: -0.15rem;
+
+    @include media-breakpoint-down(sm)
```

Comment:
This is the new bootstrap approach to deal with different devices which is quite nice. Please since you are the first one to use it, switch all other statements to this new one.